src/test/java/LoginTest.java

```diff
@@ -0,0 +1,83 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class LoginTest {
+
+    private WebDriver driver;
+
+    @BeforeSuite
+    public void setupDriverManager() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @BeforeTest
+    public void openBrowser() {
+        this.driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofMillis(3000));
+    }
+
+    @AfterTest
+    public void closeBrowser() {
+        this.driver.quit();
+    }
+
+    @Test
+    public void Login() {
+        // Launch website
+        driver.get("https://app.neuralegion.com/login");
+
+        // Get the WebElement corresponding to the Login
+        WebElement login = driver.findElement(By.id("mat-input-0"));
+
+        // Get the WebElement corresponding to the Password
+        WebElement password = driver.findElement(By.id("mat-input-1"));
+
+        login.sendKeys("catbubliktest1@yandex.ru");
```

Comment:
Please, define email and password as constants at the start of `Login` method:
```java
final String email = "catbubliktest1@yandex.ru"; 
```
And use them in methods:
```java
login.sendKeys(email);
```

We will make them as test method parameters in the future in order to test correct and incorrect credentials.